public/index.ejs

```diff
@@ -181,12 +185,26 @@ async function update() {
   if (username === '')
     return;
 
+  // Abort resources being fetched.
+  abortPendingFetches();
+
   // Get a list of all the user's repositories, and filter them based on whether they have deployments urls.
   let nextLink = `https://api.github.com/users/${username}/repos?per_page=50`;
 
   while (nextLink) {
     console.log(`Fetching nextLink=${nextLink}`);
-    let response = await fetch(nextLink, {headers: kGitHubHeaders});
+
+    // create a new AbortController and add it the list
+    const controller = new AbortController();
+    let abortControllerObject = {};
+    abortControllerObject[nextLink] = controller;
```

Comment:
Yeah, it's not being used.  I thought it might be useful to have the additional context.  I'm fine with taking it away, which would be an all or nothing situation for the AC list.

Reply:
Want me to make this change?  

The strategy that we landed on came out of later discussions, and wasn't clear initially whether this option would be useful.  Although, there is less reason to have a function to build the object if the object is simplified.  I guess that would simply be an array of AC's at that point.

Reply:
Yeah I’m mixed. I guess it makes sense to go the simpler for now since we don’t need the URL and therefore the added complexity?

Reply:
Yeah, simpler is better, when in doubt.